civismlext/test/test_stacking.py

```diff
@@ -729,6 +729,22 @@ def test_fit_regression():
     assert sr.meta_estimator.fit_params == {'bar': 'b'}
 
 
+def test_cv_shuffle_indices():
+    """Make sure xmeta and ymeta retain the correct order, even when the CV
+    generator is shuffling.
+    """
+    estlist = [('be1', PassThruReg()),
+               ('be2', PassThruReg()),
+               ('meta', PassThruReg())]
+    sr = StackedRegressor(estlist, cv=KFold(n_splits=2, shuffle=True))
```

Comment:
Maybe I'm missing something, but I'm not sure why this test passes. You don't set a random seed in `cv`. Why is the shuffle order guaranteed to be consistent with `inds` here?

Reply:
Ah, yes. The shuffle order doesn't need to be consistent with `inds`. All that matters is that the *relative* orders of `xmeta` and `ymeta` are wapped with respect to each other.

Reply:
Ok, I see what you're saying. I think this is testing the right thing, but I think it needs a documentation block to explain why. I would explain what you told me offline about two-fold cross-validation meaning that the train and test blocks will be of length 3 for each fold, and that the test block will be all indices not in the training block, and that that is what `inds` represents here.

I tried to think of a more intuitive test, but none of the problem parts of the function are stored in attributes, so we can't really test against them. I think this test will be fine if it has a big explanatory comment block somewhere.